loadScript.js

```diff
@@ -1,13 +1,19 @@
 module.exports = function loadScript (src, cb) {
   var doc = document
-  var tag = 'script'
-  var firstScript
-  var el
-  el = doc.createElement(tag)
-  firstScript = doc.getElementsByTagName(tag)[0]
-  el.async = 1
-  el.src = src
+  var isCss = src.match(/\.css$/)
```

Comment:
also, we can make it a param to the function if that's the concern. I don't see why duplicating a majority of the code and creating a whole new library is cleaner.